cloudify-manager-worker/values.yaml

```diff
@@ -306,3 +332,161 @@ containerSecurityContext:
       - SETFCAP
       - SETPCAP
       - AUDIT_WRITE
+
+# -- Parameters greoup for bitnami/rabbitmq helm chart.
+# Details: https://github.com/bitnami/charts/blob/main/bitnami/rabbitmq/README.md
+# @default -- object
+rabbitmq:
+  deploy: true
```

Comment:
@yuvlar I want to have **rabbitmq.deploy=true** by default, it's useful for new users, but this is braking changes (if old users wants to update their stack to new helm chart version, they need to add parameter **rabbitmq.deploy=false** before upgrade).
We also can  set **rabbitmq.deploy=false** by default, so old users can upgrade their stacks without any problems, but in this case new users should set  *rabbitmq.deploy=true** in their configs for install rabbit automatically.
What do you think about this?

Reply:
I think that for such changes we should do a grace period, The most accurate one in our case is a month since this is our Trial period.
Because now, even though SaaS envs from the newest blueprint will be pinned to a helm chart version, all the  deployments from previous blueprints, in case of restart for whatever reason might try to update their helm chart version and get broken.
Therefore, if you think it's ok, we can set it to false now, in a month update it to true (because eventually it is the right default for it) and update that in the changelog as changed.

Reply:
Great suggestion, changed to "false".